.gitignore

```diff
@@ -9,6 +9,7 @@
 *.egg
 *.egg-info
 dist
+dist_cext
```

Comment:
Thank you for your suggestion. I will move dist_cext to dist/cext and update my PR soon!